staging/src/k8s.io/api/storage/v1/types_swagger_doc_generated.go

```diff
@@ -28,10 +28,11 @@ package v1
 
 // AUTO-GENERATED FUNCTIONS START HERE
 var map_StorageClass = map[string]string{
-	"":            "StorageClass describes the parameters for a class of storage for which PersistentVolumes can be dynamically provisioned.\n\nStorageClasses are non-namespaced; the name of the storage class according to etcd is in ObjectMeta.Name.",
-	"metadata":    "Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata",
-	"provisioner": "Provisioner indicates the type of the provisioner.",
-	"parameters":  "Parameters holds the parameters for the provisioner that should create volumes of this storage class.",
+	"":              "StorageClass describes the parameters for a class of storage for which PersistentVolumes can be dynamically provisioned.\n\nStorageClasses are non-namespaced; the name of the storage class according to etcd is in ObjectMeta.Name.",
+	"metadata":      "Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata",
+	"provisioner":   "Provisioner indicates the type of the provisioner.",
+	"parameters":    "Parameters holds the parameters for the provisioner that should create volumes of this storage class.",
+	"reclaimPolicy": "ReclaimPolicy is the reclaim policy that dynamically provisioned PersistentVolumes of this storage class are created with",
```

Comment:
Minor nit.

Dynamically provisioned PersistentVolumes of this storage class are created with this reclaimPolicy.